INSTALL.md

```diff
@@ -12,6 +12,7 @@ dokku$ dokku git:set job-server deploy-branch main
 ```bash
 dokku config:set job-server ADMIN_USERS='xxx'
 dokku config:set job-server BASE_URL='https://jobs.opensafely.org'
+dokku config:set job-server CSP_REPORT_URI='https://xxx.ingest.sentry.io/api/xxx/security/?sentry_key=xxx&sentry_environment=xxx'
```

Comment:
Sure, although the auto-generated link at the moment doesn't include the environment, so someone will have to remember to add that manually.

Reply:
ooh, hmm.  I guess we only have one environment so it's fine, but it would be nice to have that set explicitly in case it messes up our filtering there.  More thought needed on that one I think!